homeassistant/components/islamic_prayer_times/translations/en.json

```diff
@@ -1,23 +1,23 @@
 {
-    "config": {
-        "abort": {
-            "one_instance_allowed": "Only a single instance is necessary."
-        },
-        "step": {
-            "user": {
-                "description": "Do you want to set up Islamic Prayer Times?",
-                "title": "Set up Islamic Prayer Times"
-            }
-        }
+  "config": {
```

Comment:
Why does this file change.

Reply:
Nothing changed. It just got some spacing modification I guess.

Reply:
So then we don't need to change it, right?